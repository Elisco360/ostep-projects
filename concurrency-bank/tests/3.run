./bank 300 tests/no_transactions.txt