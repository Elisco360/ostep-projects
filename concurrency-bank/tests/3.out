bank: amount file1 file2
